javascript/javascript2/week1/JS2W1.css

```diff
@@ -0,0 +1,8 @@
+#div {
+    background-color: aqua;
+    width: 260px;
+    height: 100px;
```

Comment:
Try not to give large pixel values in CSS - it's better practice to use `rem` or percentages